src/js/intlTelInput.js.ejs

```diff
@@ -429,7 +429,7 @@ Plugin.prototype = {
     var that = this;
 
     // check for cookie
-    var cookieAutoCountry = ($.cookie) ? $.cookie("itiAutoCountry") : "";
+    var cookieAutoCountry = (!!window.Cookies) ? Cookies.get("itiAutoCountry") : "";
```

Comment:
Can you tell me the reason for the double bang? I know it casts to Boolean, but that happens automatically in conditionals like this. Is it somehow safer? Or do you think it is more readable? Because I think it's less readable. (it's also inconsistent with the rest of the codebase).